contract/src/main/resources/swagger/kafbat-ui-api.yaml

```diff
@@ -3445,12 +3445,22 @@ components:
         failed_connectors_count:
           type: integer
           nullable: true
-        tasks_count:
+        tasksCount:
```

Comment:
looks like a weird mix of camelcase and snake case, why `connectors_count` but `tasksCount`?